Tensile/KernelWriterAssembly.py

```diff
@@ -349,8 +348,33 @@ def __init__( self, kernelMinNaming, kernelSerialNaming ):
     self.localReadOffsetB = 0
     self.inTailLoop = False
 
+    self.vgprOccupancy = [0]*(256+1)
+    for i in range(0,   24+1): self.vgprOccupancy[i] = 10
+    for i in range(25,  28+1): self.vgprOccupancy[i] = 9
+    for i in range(29,  32+1): self.vgprOccupancy[i] = 8
+    for i in range(33,  36+1): self.vgprOccupancy[i] = 7
+    for i in range(37,  40+1): self.vgprOccupancy[i] = 6
+    for i in range(41,  48+1): self.vgprOccupancy[i] = 5
+    for i in range(49,  64+1): self.vgprOccupancy[i] = 4
+    for i in range(65,  84+1): self.vgprOccupancy[i] = 3
+    for i in range(85, 128+1): self.vgprOccupancy[i] = 2
+    for i in range(129,256+1): self.vgprOccupancy[i] = 1
 
 
+  ########################################
+  def getOccupancy(self, kernel, vgprs):
+    multiplier = int(ceil(max(kernel["NumThreads"], 256) / 256.0))
+    # example: wg=512 multiplier=2, 1024=4
+
+    maxLds = 65536
+    ldsSize = kernel["LdsNumElements"] * kernel["ProblemType"]["DataType"].numBytes()
+    ldsLimitedOccupancy = int(ceil(maxLds / float(ldsSize)))
```

Comment:
LDS allocations have 256 byte granularity, so we should do:
ldsSize = (ldsSize + 255) & 0xff00